imaging/Pixelator.cpp

```diff
@@ -18,7 +18,7 @@ namespace imaging
   }
   bool Pixelator::prepare()
   {
-    add_color(PixelValue(255, 255, 255));
+    add_color(PixelValue(0, 0, 255));
```

Comment:
this thing absolutely  needs more visual checks and/or verification of the color choices (oos rn though)

Reply:
Created #34 .